Dockerfile

```diff
@@ -1,11 +1,16 @@
-FROM maven:3.9.1-eclipse-temurin-17
+FROM maven:3.9.1-eclipse-temurin-17 AS build
 WORKDIR /project
 
 COPY pom.xml /project/pom.xml
 COPY analyzer/pom.xml /project/analyzer/pom.xml
 RUN mvn dependency:copy-dependencies
 
 COPY analyzer/src /project/analyzer/src
-RUN mvn package
+RUN mvn package -DskipTests
```

Comment:
Tests should not be skipped. If this was added because flaky tests break running in Docker containers the tests should just be disabled all together (i.e. `@Disabled` on the flaky test case).